### ISSUE TITLE
Deprecated muleContext delegates to channel map instead of source map

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#3355]
This just wasn't correctly updated for 3.0.2 and the source map changes.

Imported Issue. Original Details:
Jira Issue Key: MIRTH-3455
Reporter: narupley
Created: 2014-09-12T14:04:24.000-0700

**Comment by rbeckman-nextgen** [comment]
Revision 7369: Updated MuleContext to use the source map rather than the channel map.

Imported Comment. Original Details:
Author: narupley
Created: 2014-09-12T14:28:30.000-0700

**Comment by rbeckman-nextgen** [comment]
I created a channel in 2.X to verify that I was able to pull out mapper values with the muleContext. I imported this channel into 3.1 and it failed to retrieve data. After slightly modifying the call from 'muleContext.getProperty()' to 'muleContext.getProperties().get()', I was able to get the destinationSet property from the source map. I also tested and verified I was able to retrieve other mapper values injected into the sourceMap by way of the new channel writer.

Imported Comment. Original Details:
Author: jaysenp
Created: 2014-09-16T11:42:07.000-0700
